### ISSUE TITLE
HTTPS mixed content warning fix Iframe API

### ISSUE THREAD

**Issue by kaha-digital** [orizens/ngx-youtube-player#7]
Great module, I noticed that when using this module on my pWA over HTTPS I get a mixed content warning, I changed this line in the file youtube-player-service.js:

        playerApiScript.src = "http://www.youtube.com/iframe_api";

to this:
`        playerApiScript.src = "//www.youtube.com/iframe_api";`

It fixes the problem (youtube iframe is now loaded via https if the request source is https

**Comment by orizens** [comment]
Should check if it works with http this way as well.

**Comment by orizens** [comment]
a new optional input property will be available in v0.3 to set the protocol to whatever you want. the default is set to "http". to set it for https would be:  
```xml
<youtube-player protocol="https"></youtube-player>
<youtube-player [protocol]="protocol"></youtube-player>
```

**Comment by kaha-digital** [comment]
That'll do the trick - cheers
